src/data/examples/exclusivity-strategy.json

```
@@ -0,0 +1,11 @@
+{
+  "default": "none",
+  "tokens": {
+    "USDT": {
+      "default": "none",
+      "destinationChains": {
+        "690": "none"
```

I'm keen to test this on Polygon in advance of having the e2e config flow up and running.

```suggestion
        "137": "weightedRandom"
```